config/exporters/monitoring/grafana/base/prometheus-exporter-service.yaml

```diff
@@ -1,95 +1,108 @@
+# Example metrics-generating service for showcasing service monitor generation. Based on:
+# https://github.com/brancz/kube-rbac-proxy/tree/master/examples/non-resource-url
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: appstudio-grafana-datasource-exporter
+  name: dummy-service-test
```

Comment:
this change is not required, you can name it appstudio-grafana-datasource-exporter

Reply:
@Kousalya1998 there are still multiple mentions of `dummy` inside the yamls.
Please rename everything so it reflects its use instead of dummy-something.